packages/transit-vehicle-overlay/package.json

```diff
@@ -9,13 +9,10 @@
   "module": "esm/index.js",
   "private": false,
   "dependencies": {
-    "@opentripplanner/base-map": "^1.0.4",
```

Comment:
This is interesting, Storybook seems to display the map fine for this package's stories, even without this import.